localnet/Makefile

```diff
@@ -29,6 +30,13 @@ upload:
 test:
 	bash ./scripts/run.sh -k
 
+test-docker:
+ifndef HARMONY_PATH
```

Comment:
why ask set another env variable? just rely on GOPATH as done in cmd:
```
docker run -it -v "$(go env GOPATH)/src/github.com/harmony-one/harmony:/go/src/github.com/harmony-one/harmony" harmonyone/localnet-test
```

Reply:
You are right. I made this because I was thinking about running the tests with docker without mounting the local folder too. But I will remove that and just care about GOPATH. Thanks!

Reply:
> You are right. I made this because I was thinking about running the tests with docker without mounting the local folder too. But I will remove that and just care about GOPATH. Thanks!

can you update the command? then we can merge this. thanks.

Reply:
Hi @gupadhyaya I pushed the requested change. Let me know if you need anything else. Thanks!